terraform/aws/variables.tf

```diff
@@ -109,3 +109,8 @@ variable "amis" {
     us-west-2 = "ami-00657261"
   }
 }
+
+variable "docker_version" {
+  description = "Docker version"
+  default = "1.8.2-0~trusty"
```

Comment:
Agreed - we could then build the 1.9 into https://github.com/Capgemini/Apollo/pull/520 on top of this change